net.worldofpadman.WoP.desktop

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
-Name=World of Padman
-Comment=Comic 3D-Shooter
+Name=World of PADMAN
```

Comment:
I see that the [Snap version](https://snapcraft.io/worldofpadman) also has "Padman" instead of the whole word in uppercase. The same thing is also on [IndieDB](https://www.indiedb.com/games/world-of-padman) and also on some parts of the [official website](https://worldofpadman.net/en/). Was the game name changed or are both variants used? Thanks! :-)

Reply:
Actually the name is World of PADMAN but often written as World of Padman. The main character is PADMAN, not Padman. However, we try to unify this in future.

Reply:
Ah, thanks for the info! :-)